### PR TITLE
DEPLOY-629: ADD missing KubernetesCluster CFN tags

### PR DESCRIPTION
**Pull request by svidrascu**

### CODE REVIEW

**Comment by svidrascu** [comment]
Any chance of getting this trough? @amohammedalfresco @mmuller88 ?

**Comment by derekhulley** [comment]
It's been approved.
Check the builds, test and merge please @svidrascu.

File: templates/bastion-and-eks-cluster.yaml [hunk 1]
```diff
@@ -200,6 +203,10 @@ Parameters:
     EksExternalUserArn:
       Type: String
       Description: The AWS IAM user arn who will be authorised to connect the cluster externally
+    EKSClusterName:
```
[the remark this answers is not in the document]
Comment (svidrascu) [reply templates/bastion-and-eks-cluster.yaml]: The default is only used when the bastion stack is ran alone. But yea, shouldnt be a problem to remove the default, ok.

**Comment by svidrascu** [comment]
I am closing this PR in favor of https://github.com/Alfresco/acs-deployment-aws/pull/22 so i can get some builds going also.

**Pull request closed without merging**